<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorTab</class>
 <widget class="QWidget" name="EditorTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>581</width>
     <height>251</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>落脚点</string>
    </attribute>
    <widget class="QPushButton" name="normalLandButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>101</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>普通地面</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="widthInput">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>140</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>50</number>
     </property>
     <property name="maximum">
      <number>900</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="heightInput">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>180</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>50</number>
     </property>
     <property name="maximum">
      <number>900</number>
     </property>
    </widget>
    <widget class="QPushButton" name="spikeLandButton">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>141</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>地刺</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="attackInput">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>180</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
    <widget class="QPushButton" name="checkPointButton">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>20</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>检查点</string>
     </property>
    </widget>
    <widget class="QPushButton" name="movingLandButton">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>110</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>移动地板</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="landTInput">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>190</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
    </widget>
    <widget class="QPushButton" name="springLandButton">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>120</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>弹跳地板</string>
     </property>
    </widget>
    <widget class="QPushButton" name="fragileButton">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>180</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>易碎方块</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>敌人</string>
    </attribute>
    <widget class="QPushButton" name="worm1Button">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>110</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>普通源石虫</string>
     </property>
    </widget>
    <widget class="QPushButton" name="worm2Button">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>130</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>酸液源石虫</string>
     </property>
    </widget>
    <widget class="QPushButton" name="worm3Button">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>160</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>高能源石虫</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>陷阱</string>
    </attribute>
    <widget class="QPushButton" name="worm1GeneratorButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>111</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>源石虫生成器</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="tInput">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>150</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
    </widget>
    <widget class="QPushButton" name="worm2GeneratorButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>50</y>
       <width>141</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>酸液源石虫生成器</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>道具</string>
    </attribute>
    <widget class="QPushButton" name="mushroomButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>生命蘑菇</string>
     </property>
    </widget>
    <widget class="QPushButton" name="flowerButton">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>70</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>技能花</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLineEdit" name="mapName">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>300</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="moveToLeft">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>0</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="moveToRight">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>0</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>300</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
